src/modules/areas.js

```diff
@@ -4,7 +4,7 @@ import { API_BASE_URL } from '../constants';
 // Actions
 const GET_USER_AREAS = 'areas/GET_AREAS';
 const SET_LOADING_AREAS = 'areas/SET_LOADING_AREAS';
-const SET_LOADING_ERROR = 'areas/SET_LOADING_ERROR';
+const SET_LOADING_AREAS_ERROR = 'areas/SET_LOADING_AREAS_ERROR';
```

Comment:
We discussed abstracting this to a universal notification component, possibly in the form of a single object outside the areas, reports, etc state. Just commenting so we remember to discuss.